src/main/java/com/scalefocus/libraryproject/controllers/UserController.java

```diff
@@ -3,9 +3,7 @@
 import com.scalefocus.libraryproject.models.UserModel;
 import com.scalefocus.libraryproject.services.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please, don't use wildcard imports. This is a bad practice in java development and may cause security problems.